### PR TITLE
fix(badge): default size if wrong value is given

### PR DESCRIPTION
**Pull request by matthprost**
## Type

- Enhancement

#### What is expected?

If prop given doesn't exists we still set as medium as default size to avoid strange behavior.

### CODE REVIEW

File: src/components/Badge/index.tsx [hunk 1]
```diff
@@ -131,7 +131,7 @@ const Badge: FunctionComponent<BadgeProps> = ({
     [prominence, theme],
   )
 
-  const sizeValue = SIZES[size]
+  const sizeValue = SIZES[size] || SIZES.medium
```
Comment (chambo-e) [inline src/components/Badge/index.tsx:134]:
```suggestion
  const sizeValue = SIZES?.[size] ?? SIZES.medium
```

?

Comment (chambo-e) [reply src/components/Badge/index.tsx:134]: Can we also ad a test ?

Comment (matthprost) [reply src/components/Badge/index.tsx:134]: more like `SIZES[size] ?? SIZES.medium` as `SIZES` always exists, I will add testing :)

Comment (matthprost) [reply src/components/Badge/index.tsx:134]: more like `SIZES[size] ?? SIZES.medium` as `SIZES` always exists, I will add testing :)

**Pull request closed without merging**